scripts/setup/constraints.txt

```diff
@@ -289,7 +291,7 @@ pyserial==3.5
     #   bflb-iot-tool
     #   mbed-os-tools
     #   mbed-tools
-pytest==6.2.5 ; platform_machine != "aarch64" and sys_platform == "linux"
+pytest==6.2.5 ; sys_platform == "linux"
```

Comment:
I think we can leave only this line changed in this file, the rest can be reverted.